groups/script.js

```diff
@@ -13,6 +13,8 @@ const sections = document.querySelectorAll('.manage-groups, .create-group');
 const loader = document.querySelector('.backdrop');
 const userIsNotVerifiedText = document.querySelector('.not-verified-tag');
 const userSelfData = await getUserSelf();
+const params = new URLSearchParams(window.location.search);
+const isDev = params.get('dev');
```

Comment:
lets match its value to be true 
else tomorrow I can just write dev='someRandomStuff" and is dev will still be considered